ros_buildfarm/sourcedeb_job.py

```diff
@@ -108,7 +108,14 @@ def build_sourcedeb(sources_dir, os_name=None, os_code_name=None):
         '--git-ignore-branch',
         # dpkg-buildpackage args
         '-S']
-    if os_name == 'debian' and os_code_name == 'stretch':
+    ubuntu_before_artful = (
+        'precise', 'quantal', 'raring', 'saucy',
+        'trusty', 'utopic', 'vivid', 'wily',
+        'xenial', 'yakkety', 'zesty')
+    if (
+        os_name == 'debian' and os_code_name == 'stretch' or
```

Comment:
Before modification for debian would make sense so this doesn't bit us in the future on debian.

Reply:
Done in 4363beef3a3c621470fa582fa95ae15f1090824a.